connectors-home/modules/ROOT/pages/introduction/connectors-shared-responsibilities.adoc

```diff
@@ -0,0 +1,90 @@
+
+= Anypoint Connectors Shared Responsibilities
+
+The successful operation of Anypoint Connectors is a shared responsibility between MuleSoft and the customer. It is critical to understand which areas customers are responsible for and which areas MuleSoft is responsible for.
+
+== Anypoint Connectors Categories
+
+Anypoint Connectors fall into these categories:
+
+* Connectors that enable the use of cross-platform protocols or open APIs. +
+For example, Database Connector, which is based on Oracle’s JDBC protocol.
+
+* Connectors or modules that enable connectivity to MuleSoft or Salesforce applications. +
+For example, Salesforce Marketing Cloud Connector and Anypoint MQ Connector.
+
+* Connectors that enable connectivity to third-party or partner APIs. +
+For example, Azure Key Vault Connector and Stripe Connector.
+
+* MuleSoft Certified Connectors +
+These are developed by MuleSoft’s partners and subsequently reviewed and certified by MuleSoft, for example ESRI Connector (in Anypoint Exchange identify them as MuleSoft Partner).
+
+* Community Connectors +
+These are developed by the developer community and subsequently reviewed and certified by MuleSoft, for example, Slack Connector - Mule 3 (in Anypoint Exchange identify them as Open-Source Community).
+
+== Third-party Providers and Partners
+
+Additionally, there are third-party providers and partners:
+
+* Third-party providers +
+These entities, which are external to the MuleSoft organization, provide and develop libraries or drivers that you add and configure for the operation of the connector under certain circumstances. MuleSoft does not include these libraries or drivers in the connector source code. 
++
+For example, Database Connector connects through a common protocol (JDBC) with a database. There are several drivers that offer a JDBC interface, which is a communication channel between the connector and the database. You select which JDBC driver you add and configure to connect to a specific vendor’s database.
```

Comment:
```suggestion
For example, Database Connector uses a common protocol (JDBC) to connect to a database. There are several drivers that offer a JDBC interface, which is a communication channel between the connector and the database. You add and configure which JDBC driver to use for connecting to a specific vendor’s database.
```

Reply:
I tried to tighten the text above by removing "which is a communication channel between the connector and the database," since it isn't necessary to say that if you reword the first sentence. I am not sure I got the wording right in terms of accuracy.